components/library/zeromq/zeromq5/zeromq.p5m

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
I refer to all versions available before this integration. When you add next version, you'll also add it to corresponding zeromq meta-package.